qa/pull-tester/rpc-tests.py

```diff
@@ -194,6 +195,12 @@ def main():
             # (for parallel running efficiency). This combined list will is no
             # longer sorted.
 
+    # Remove the test cases that the user has explicitly asked to exclude.
+    if args.exclude:
+        [ test_list.remove(exclude_test + ".py")
```

Comment:
ha. If I write for loops I get review comments saying I should use list comprehensions, and if I write list comprehensions I get review comments saying I should use for loops :)

In this case you're probably right that the for loop is clearer.